Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM alpine:latest
+FROM rust:1.41 as builder
+WORKDIR /app 
+COPY . /app
+RUN cargo build --release  && \
+    chmod +x target/release/dotenv-linter
 
-COPY dotenv-linter /usr/local/bin/
+FROM scratch
 
-ENTRYPOINT ["dotenv-linter"]
+COPY --from=0 /app/target/release/dotenv-linter /
```

Comment:
Being a bit picky here, but wouldn't be `COPY --from=builder` a bit more explicit here and more readable?
That is understandable instantly.
I just assume that this copys from the first (zero indexed) build image and 1 would be the resulting image?

Otherwise, great job, I like it :)

Reply:
old habit, fixed